modules/security/src/test/java/io/oasp/module/security/common/impl/accesscontrol/AccessControlConfigTest.java

```diff
@@ -0,0 +1,117 @@
+package io.oasp.module.security.common.impl.accesscontrol;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import javax.annotation.security.RolesAllowed;
+
+import org.junit.Test;
+
+import io.oasp.module.security.common.api.accesscontrol.AccessControl;
+import io.oasp.module.security.common.api.accesscontrol.AccessControlGroup;
+import io.oasp.module.security.common.base.accesscontrol.AccessControlConfig;
+import io.oasp.module.test.common.base.ModuleTest;
+
+/**
+ * Test of {@link AccessControlConfig}.
+ */
+public class AccessControlConfigTest extends ModuleTest {
+
+  @Test
+  // @RolesAllowed only used to ensure that the constant can be referenced here
+  @RolesAllowed(AccessControlConfigSimple.PERMISSION_FIND_TABLE)
+  public void testSimple() {
+
+    // given + when
+    AccessControlConfigSimple config = new AccessControlConfigSimple();
+    AccessControlGroup chief = (AccessControlGroup) config.getAccessControl(AccessControlConfigSimple.GROUP_CHIEF);
+
+    // then
+    assertThat(chief).isNotNull();
+    assertThat(chief.getId()).isEqualTo(AccessControlConfigSimple.GROUP_CHIEF);
+    assertThat(flatten(chief.getPermissions())).containsExactly(AccessControlConfigSimple.PERMISSION_SAVE_OFFER,
```

Comment:
Why do they have to be in order?
I think this should be `InAnyOrder` as well as below.

Reply:
I am fine with that and will change this accordingly.